index.js

```
@@ -193,6 +193,39 @@ module.exports = function drivex(driver, wd) {
       }, timeout || 5000).then(function() {
         return keyFound;
       });
+    },
+    /**
+     *validateText validates the text for a WebElement
+     * @param locator
+     * @param parentWebElement (optional)
+     * @param expected text
+     * @returns {*}
+     */
+    validateText: function (locator, parentWebElement, expectedText) {
+      return methods.find(locator, parentWebElement,expectedText).getText().then(function (actual) {
```

Instead of resolving as "false" we should throw an error in order to be consistent with other similar functions.